docs/source-pytorch/advanced/training_tricks.rst

```diff
@@ -78,18 +78,24 @@ Auto-scaling of batch size can be enabled to find the largest batch size that fi
 memory. Large batch size often yields a better estimation of the gradients, but may also result in
 longer training time. Inspired by https://github.com/BlackHC/toma.
 
-.. seealso:: :class:`~pytorch_lightning.trainer.trainer.Trainer`
+.. seealso:: :class:`~pytorch_lightning.tuner.tuning.Tuner`
 
 .. code-block:: python
 
-    # DEFAULT (ie: don't scale batch size automatically)
-    trainer = Trainer(auto_scale_batch_size=None)
+    from pytorch_lightning.tuner import Tuner
+
+    # Create a tuner for the trainer
+    trainer = Trainer(...)
+    tuner = Tuner(trainer)
 
-    # Autoscale batch size
-    trainer = Trainer(auto_scale_batch_size=None | "power" | "binsearch")
+    # Auto-scale batch size by growing it exponentially (default)
+    tuner.scale_batch_size(model, mode="power")
```

Comment:
how about rather `maximize_batch_size` as scaling is up and down...
```suggestion
    tuner.maximize_batch_size(model, mode="power")
```

Reply:
@Borda Let's create an issue for this since this would be out of scope for the PR :)

Reply:
#16529